contracts/cheddar_5x5_tic_tac_toe/src/internal.rs

```diff
@@ -139,8 +143,10 @@ impl Contract {
                 Some(amount) => amount,
                 None => panic!("Failed divide deposit to refund GameDeposit (GameResult::Tie)"),
             };
+            print!("refund ammount {}", refund_amount);
+            print!("reward.balance.0 {}", reward.balance.0);
             assert!(
```

Comment:
```suggestion
            require!(
```

Reply:
we should use `require!` rather than `assert!` if possible - it's a bit more faster.